napari_allencell_annotator/util/file_utils.py

```diff
@@ -51,7 +51,7 @@ def get_files_in_dir(dir_path: Path) -> list[Path]:
         dir_path: list[Path]
             The path to a directory
         """
```

Comment:
adding a comment saying we return a sorted list and/or changing the method name to reflect that would be a good touch here

Reply:
renamed and added comment in https://github.com/aics-int/napari-allencell-annotator/pull/69/commits/b26b4f22a91968b80328bfe17e55c68bc2d81fe8